_includes/comments.html

```diff
@@ -0,0 +1,60 @@
+<div class="row">
+    <div class="center">
+        <div class="col-md-3">
+            <div class="center">
+                <h2 class="no-anchor">Comments </h2>
+            </div>
+            <div class="center share-buttons">
+                <ul>
+                    {% capture url %}{{site.url}}{{page.url}}{% endcapture %}
+                    <li>
+                        <a  href="https://www.facebook.com/sharer/sharer.php?u={{ url }}"
```

Comment:
What about page title and description? Those can be pre-filled in the tweet/FB share/reddit share/etc too, right?

Reply:
Also, I think you will need to URL-encode those parameters. E.g., `u={{ url | url_encode }}`.